lib/services/local/users.go

```diff
@@ -1491,6 +1491,20 @@ func (s *IdentityService) GetKeyAttestationData(ctx context.Context, publicKey c
 	return &resp, nil
 }
 
+func fingerprintSha256(pubDER []byte) string {
+	uuid := uuid.NewHash(crypto.SHA256.New(), uuid.UUID{}, pubDER, 0)
```

Comment:
crypto.SHA256 is a bit special, in that you need to import crypto/sha256 _somewhere_ for it to work.

We certainly do in Teleport, but the "correct" is to `import _ crypto/sha256` in the files where it is used.

https://pkg.go.dev/crypto#SHA256

Reply:
Good to know! will add.